docs/SuccessFactors-connector.md

```diff
@@ -0,0 +1,23 @@
+# SuccessFactors Connection
+
+Description
+-----------
+Use this connection to access data in ServiceNow.
+
+Properties
+----------
+**Name:** Name of the connection. Connection names must be unique in a namespace.
+
+**Description:** Description of the connection.
+
+**SAP SuccessFactors Logon Username (M)**: SAP SuccessFactors Logon Username for user authentication.  
+
+**SAP SuccessFactors Logon Password (M)**: SAP SuccessFactors Logon password for user authentication.
+
+**SAP SuccessFactors Base URL (M)**: SAP SuccessFactors Base URL.
+
+* **JSON**: Contents of the service account JSON file.
```

Comment:
Why is JSON indented?

Reply:
Removed the line. It's being added by mistake.